src/dev/MAX31855.cpp

```diff
@@ -1,27 +1,69 @@
 #include <EVT/io/SPI.hpp>
 #include <dev/MAX31855.hpp>
+#include <utility>
 
 using namespace EVT::core::IO;
 
 namespace TMU::DEV {
 
 MAX31855::MAX31855(SPI& spi, uint8_t device) : spi(spi), device(device) {}
 
-uint16_t MAX31855::readTemp() {
-    uint16_t temp;
+MAX31855::MaxStatus MAX31855::readTemp(uint16_t& temp) {
+    uint8_t lastDataBit = 0;
+    uint16_t returned_temp = 0;
 
     const uint8_t length = 4;
-    uint8_t bytes[length];
+    uint8_t bytes[length] = {'\0'};
```

Comment:
Why did you add this?

Reply:
I just don't want any crummy data when the array gets initialized, its just a habit of mine.